config.json

```diff
@@ -46,6 +46,17 @@
         "Transforming"
       ]
     },
+	{
+      "uuid": "2ee3cc7a-db3f-4668-9983-ed6d0fea95d1",
+      "slug": "two-fer",
+      "core": true,
```

Comment:
@ErikSchierboom which exercises did you think this should unlock? I got the inspiration for the fact that this was a core exercise from the C# track, but didn't think to look at what exercises (if any) it should unlock.